src/data/plugins/steamflow/plugin.md

```
@@ -0,0 +1,66 @@
+---
+id: A7F3C4E16B8D4B5AB2D7139EEC8FA0B4
+tags:
+- Gaming
+- Application Launchers
+---
+
+## Control Steam from Flow Launcher
+![](images/peenar.gif)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)